lib/kube/kubeconfig/kubeconfig.go

```diff
@@ -114,6 +124,9 @@ func Update(path string, v Values) error {
 			if v.Exec.TshBinaryInsecure {
 				authInfo.Exec.Args = append(authInfo.Exec.Args, "--insecure")
 			}
+			if envVars != nil {
```

Comment:
I would use `if len(envVars) > 0` as it will work for both a nil slice and a non-nil slice with zero elements.